frontend/src/components/PartsOfSpeech.vue

```diff
@@ -24,30 +25,69 @@ export default {
   },
   created() {
     var posTagger = require("wink-pos-tagger");
+    '<div class = "msg">';
     if (this.$store.state.responseScore)
       this.conversation = this.$store.state.conversation;
-
+    '</div>';
     var i;
     var f;
```

Comment:
Why not initialise inside the loop? It does not seem like you are using it other than looping.